### PR TITLE
Create Staff View login page

### LINKED ISSUE #516: Login page
- Add RetroCertsStaffViewAuthPage to `/src/client/pages`. This page should be modeled off of RetroCertsAuthPage with the following differences: 
  - It shouldn't include a ReCaptcha.

Design specs can be found here: https://www.figma.com/file/voyh0E7h1WmzvpUWs8bL0g/CA-UI-%E2%80%93%C2%A0Retroactive-Certification-%E2%80%93%C2%A0UI?node-id=0%3A1

### PR DESCRIPTION
**Pull request by kalvinwang**
This PR creates the view for the Staff View login page. 

This page is identical to RetroCertsAuthPage, minus the captcha, Spanish translation button, Show SSN button, and helper text below the header. It also has a different error message for userNotFound and submit button copy.

image

===

Resolves #516

- [x] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: src/client/pages/StaffViewAuthPage/index.js [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+        }
+      })
+      .catch((error) => console.error(error));
+  };
+
+  const handleChange = (event, setter) => {
+    setter(event.target.value);
+  };
+
+  useEffect(() => {
```
Comment (sharonwarner) [inline src/client/pages/StaffViewAuthPage/index.js:139]: Let's confirm with @nguyen-tina, but I believe we don't want to hide the SSN. We can just use a regular text input.

Comment (nguyen-tina) [reply src/client/pages/StaffViewAuthPage/index.js:139]: That's correct!

File: src/client/pages/StaffViewAuthPage/index.test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  it("retro certs auth page with user not found error", async () => {
+    const wrapper = renderNonTransContent(Component, "StaffViewAuthPage", {
+      userData: { status: AUTH_STRINGS.statusCode.userNotFound },
+    });
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it("retro certs auth page with session timeout", async () => {
```
Comment (idealisms) [inline src/client/pages/StaffViewAuthPage/index.test.js:20]: Can we remove this test? This error message shouldn't show up.

File: src/client/pages/StaffViewAuthPage/index.js [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+  );
+
+  const status = userData && userData.status;
+  const errorTransKey = new Map([
+    [
+      AUTH_STRINGS.statusCode.userNotFound,
+      "retrocert-login.invalid-user-error-staff-view",
+    ],
+    [
+      AUTH_STRINGS.statusCode.sessionTimedOut,
```
Comment (idealisms) [inline src/client/pages/StaffViewAuthPage/index.js:49]: This won't be used on this page, but maybe you're keeping it to reduce the differences with RetroCertsAuthPage?

File: src/client/pages/StaffViewAuthPage/index.js (hunk 1 above)
Comment (kalvinwang) [reply src/client/pages/StaffViewAuthPage/index.js:139]: great catch, thanks!

File: src/client/pages/StaffViewAuthPage/index.test.js (hunk 2 above)
Comment (kalvinwang) [reply src/client/pages/StaffViewAuthPage/index.test.js:20]: oh right, good point!

File: src/client/pages/StaffViewAuthPage/index.js (hunk 3 above)
Comment (kalvinwang) [reply src/client/pages/StaffViewAuthPage/index.js:49]: nope not intentional, thanks

**Pull request merged**